gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/HttpGremlinEndpointHandler.java

```diff
@@ -189,6 +191,12 @@ public void channelRead(final ChannelHandlerContext ctx, final Object msg) {
             try {
                 logger.debug("Processing request containing script [{}] and bindings of [{}] on {}",
                         requestArguments.getValue0(), requestArguments.getValue1(), Thread.currentThread().getName());
+                if (settings.enableAuditLog) {
```

Comment:
I haven't thought this through at all but is it desirable that we would double-log here if the user has somehow used both settings (the new and deprecated one as `true`)? I see this sort of code in the `AbstractEvalOpProcessor` as well. Perhaps setting both to `true` should be considered an "illegal" configuration or the setting of the new one to `true` should override the old to `false`? Perhaps, setting the old to `true` and the new to `false` should result in a WARN to the logger to let them know they are using a deprecated settings?

Reply:
Agreed. I thought about this, but this check ends up in the gremlin server startup code, which I had not touched yet... Good to see that you are not afraid to touch your own code!

Reply:
Settings.java now logs an appropriate warning.